src/game/game.cpp

```diff
@@ -0,0 +1,40 @@
+#include "game.hpp"
+
+using namespace std;
+using namespace chrono;
+
+Game::Game(int64_t updatesPerSecond)
+{
+  this->updatesPerSec = updatesPerSecond;
```

Comment:
Instead of a constructor body, it's a good idea to use an initialization list. It makes no difference in this particular case, but it is required sometimes (const members, for example) and removes unnecessary constructor calls (see links).

Personally I have gone so far that I don't use the "regular" constructor for member initialization. Only when I need to do something programmatically there.

Some reading and examples:
https://en.cppreference.com/w/cpp/language/initializer_list
https://www.geeksforgeeks.org/when-do-we-use-initializer-list-in-c/

Reply:
:ok_hand: